tonic/models/vic/vic.py

```diff
@@ -51,7 +84,7 @@ def run(self, global_param, logdir=None):
             with open(global_param_file, mode='w') as f:
                 f.write(global_param)
 
-        self._call_vic('-g', global_param_file)
+        self._call_vic('-g', global_param_file, **kwargs)
 
```

Comment:
Yeah, I don't think this is part of your PR - Joe and I made these changes before.